### ISSUE TITLE
0.16.0 autotick bot error

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/watchfiles-feedstock#8]
### Comment:

0.16.0 just came out yesteday, but from the [status page](https://conda-forge.org/status/):

```
[watchfiles](https://github.com/conda-forge/watchfiles-feedstock/blob/master/recipe/meta.yaml): click for details

We found a problem parsing the recipe for version '0.16.0': 

DuplicateKeyError('while constructing a mapping',   in "", line 38, column 3:
      requires:
      ^ (line: 38), 'found duplicate key "commands" with value "[\'watchfiles -h\']" (original value: "[\'pip check\']")',   in "", line 44, column 3:
      commands:
      ^ (line: 44), '\n                    To suppress this check see:\n                        http://yaml.readthedocs.io/en/latest/api.html#duplicate-keys\n                    ', '                    Duplicate keys will become an error in future releases, and are errors\n                    by default when using the new API.\n                    ')

traceback:
  File "/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/migrators/version.py", line 509, in migrate
    cmeta = CondaMetaYAML(fp.read())
  File "/home/runner/work/autotick-bot/autotick-bot/cf-scripts/conda_forge_tick/recipe_parser/_parser.py", line 494, in __init__
    self.meta = self._parser.load("".join(lines))
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/main.py", line 434, in load
    return constructor.get_single_data()
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 121, in get_single_data
    return self.construct_document(node)
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 131, in construct_document
    for _dummy in generator:
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 1569, in construct_yaml_map
    self.construct_mapping(node, data, deep=True)
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 1452, in construct_mapping
    value = self.construct_object(value_node, deep=deep)
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 154, in construct_object
    data = self.construct_non_recursive_object(node)
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 196, in construct_non_recursive_object
    for _dummy in generator:
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 1569, in construct_yaml_map
    self.construct_mapping(node, data, deep=True)
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 1453, in construct_mapping
    if self.check_mapping_key(node, key_node, maptyp, key, value):
  File "/usr/share/miniconda3/envs/run_env/lib/python3.9/site-packages/ruamel/yaml/constructor.py", line 294, in check_mapping_key
    raise DuplicateKeyError(*args)
```

**Comment by davidbrochart** [comment]
Looks like the source distribution is not included in PyPI for v0.16.0, I opened https://github.com/samuelcolvin/watchfiles/issues/175.

**Comment by samuelcolvin** [comment]
Looks like there might be an error in the error processing here then that anaconda might want to know about.
